src/main/java/com/github/alexdlaird/ngrok/protocol/CreateTunnel.java

```diff
@@ -86,6 +93,13 @@ private CreateTunnel(final Builder builder) {
         this.remoteAddr = builder.remoteAddr;
         this.metadata = builder.metadata;
         this.schemes = builder.schemes;
+        if(!isNull(builder.provider)) {
+            this.oauth = new OAuth();
```

Comment:
Pass in an `OAuth` object instead so we set a single value here, no need to check for `null`.

Reply:
Done.